File: pgm/ABC_Kotlin/ABC_386_A.kt
// import java.time.*

// import kotlin.math.*

const val Yes: String = "Yes"
const val No: String = "No"
const val one: Int = 1
const val two: Int = 2
const val three: Int = 3
const val thirteen: Int = 13
const val zero: Int = 0
const val space: String = " "

fun main() {
    val (A: Int, B: Int, C: Int, D: Int) = readln().split(space).map { it.toInt() }
    val cards: IntArray = IntArray(thirteen)
    cards[A - one]++
    cards[B - one]++
    cards[C - one]++
    cards[D - one]++
    var threeCards = zero
    var pairs = zero
    for (count: Int in cards) {
        if (count == three) {
            threeCards++
            break
        } else if (count == two) {
            pairs++
        }
    }
    if (threeCards == one) {
        println(Yes)
    } else if (pairs == two) {
        println(Yes)
    } else {
        println(No)
    }
    kotlin.system.exitProcess(0)
}

File: pgm/ABC_Dart/ABC_386_A.dart
// import 'dart:collection';
import 'dart:io';
// import 'dart:math';

void main() {
  final [int A, int B, int C, int D] = stdin
      .readLineSync()!
      .split(' ')
      .map(int.parse)
      .toList();
  const String Yes = 'Yes';
  const String No = 'No';
  const int one = 1;
  const int two = 2;
  const int three = 3;
  const int thirteen = 13;
  const int zero = 0;
  final List<int> cards = List.filled(thirteen, zero);
  cards[A - one]++;
  cards[B - one]++;
  cards[C - one]++;
  cards[D - one]++;
  int threeCards = zero;
  int pairs = zero;
  for (int count in cards) {
    if (count == three) {
      threeCards++;
      break;
    } else if (count == two) {
      pairs++;
    }
  }
  if (threeCards == one) {
    print(Yes);
  } else if (pairs == two) {
    print(Yes);
  } else {
    print(No);
  }
  exit(0);
}
